assets/global.js

```diff
@@ -527,13 +527,13 @@ class SliderComponent extends HTMLElement {
     this.currentPage = Math.round(this.slider.scrollLeft / this.sliderLastItem.clientWidth) + 1;
 
     if (this.currentPage === 1) {
-      this.prevButton.setAttribute('disabled', true);
+      this.prevButton.setAttribute('disabled', 'disabled');
```

Comment:
Any suggestions on what to use here? 

I opted for `'disabled'` as it seems to be valid and more explicit than an empty string `''`, which I believe can be confusing for quickly glancing at the code, but I'm open to suggestions :)

Reply:
Yeah it doesn't seem like we can add an attribute without a value. With `disabled` as the value it seems good 👍